apps/omg_eth/test/fixtures.exs

```diff
@@ -29,13 +29,19 @@ defmodule OMG.Eth.Fixtures do
   @test_erc20_vault_id 2
 
   deffixture eth_node do
-    if Application.get_env(:omg_eth, :run_test_eth_dev_node, true) do
-      {:ok, exit_fn} = DevNode.start()
+    case System.get_env("DOCKER_GETH") do
```

Comment:
in mix_based_childchain I set an env var so that I avoid starting and stoping geth